config/crd/bases/cyndi.cloud.redhat.com_cyndipipelines.yaml

```diff
@@ -1,38 +1,13 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.5.0
```

Comment:
Do you need this for cyndi to generate correctly? Trying to determine if you have a different version or if this was an intended version bump.